### PR TITLE
add tldjs@1.7.0 with npm auto-update

### PR DESCRIPTION
**Pull request by kennynaoh**
PR for #8943 
@pvnr0082t, @x09326 please help me review this pull request, thank you.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.
# Profile of the lib
- Git repository (required): https://github.com/oncletom/tld.js
- Official website (optional, not the repository):
- NPM package url (optional): https://www.npmjs.com/package/tldjs
- GitHub / Bitbucket popularity (required):
  - Count of stars: 165
  - Count of watchers: 11
  - Count of forks: 27
- NPM download stats (optional):
  - Downloads in the last day: 791
  - Downloads in the last week: 8,211
  - Downloads in the last month: 40,457
# Essential checklist
- [ ] I'm the author of this library
  - [ ] I would like to add link to the page of this library on CDNJS on website / readme
- [x] This lib was not found on cdnjs repo
- [x] No already exist / duplicated issue and PR
- [x] The lib has notable popularity
  - [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  - [x] More than 500 downloads stats per month on npm registry
- [ ] Project has public repository on famous online hosting platform (or been hosted on npm)
# Auto-update checklist
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.
# Git commit checklist
- [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
- [x] The parent of the commit(s) in the PR is not old than 3 days.
- [x] Pull request is sending from a non-master branch with meaningful name.
- [x] Separate unrelated changes into different commits.
- [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
- [x] Close corresponding issue in commit message
- [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/tldjs/1.6.2/tld.js [hunk 1]
```diff
@@ -0,0 +1,348 @@
+"use strict";
+
+var Rule = require('./rule.js');
```
Comment (pvnr0082t) [inline ajax/libs/tldjs/1.6.2/tld.js]: I think we need `rule.js`. What do you think? @kennynaoh

**Comment by pvnr0082t** [comment]
@kennynaoh I didn't see any version lost on npm. Do I miss something?

**Comment by kennynaoh** [comment]
@pvnr0082t 
Sorry for wrong commit.
I have fixed it and added dependency file.
Please help me review it again.
Thanks!

**Comment by pvnr0082t** [comment]
LGTM

**Comment by PeterDaveHello** [comment]
Is there any reference about the "rule.js"?

**Comment by kennynaoh** [comment]
@PeterDaveHello 
Sorry I don't get it.
What you mean reference?

**Comment by PeterDaveHello** [comment]
How do you know that we need that file?

**Comment by kennynaoh** [comment]
@PeterDaveHello Is this correct? https://github.com/oncletom/tld.js/blob/master/lib/tld.js#L3

**Comment by PeterDaveHello** [comment]
Isn't that a `nodejs` require?

**Comment by kennynaoh** [comment]
@PeterDaveHello So I think I should exclude `rule` file in this case, right? Thanks!

**Comment by PeterDaveHello** [comment]
Don't know, I think the file we'll host will look like https://wzrd.in/standalone/tldjs

**Comment by PeterDaveHello** [comment]
Ping

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Comment by kennynaoh** [comment]
@PeterDaveHello Sorry for late reply. So is it better to combine those two file `lib/tld.js` and `lib/rule.js` into one? Thanks!

**Comment by PeterDaveHello** [comment]
No we don't do that by our side, why don't you keep the discussion with upstream?

**Comment by sufuf3** [comment]
@PeterDaveHello I don't think this lib can use in web front-end because this lib didn't provide the file which can use in front-end.
Can we close this PR?
Thank you.

**Comment by PeterDaveHello** [comment]
I'll manually build it.

**Comment by PeterDaveHello** [comment]
Go back to #8943

**Pull request closed without merging**